tests/manage/z_cluster/nodes/test_node_replacement_proactive.py

```diff
@@ -61,35 +59,20 @@ def test_nodereplacement_proactive(self, pvc_factory, pod_factory, dc_pod_factor
                 cephfs_dc_pod = dc_pod_factory(interface=constants.CEPHFILESYSTEM, node_name=worker_node, size=20)
                 pod.run_io_in_bg(cephfs_dc_pod, expect_to_fail=False, fedora_dc=True)
 
-        # Unscheduling node
-        node.unschedule_nodes([osd_node_name])
-        # Draining Node
-        node.drain_nodes([osd_node_name])
-        log.info("Getting machine name from specified node name")
-        machine_name = machine.get_machine_from_node_name(osd_node_name)
-        log.info(f"Node {osd_node_name} associated machine is {machine_name}")
-        log.info(f"Deleting machine {machine_name} and waiting for new machine to come up")
-        machine.delete_machine_and_check_state_of_new_spinned_machine(machine_name)
-        new_machine_list = machine.get_machines()
-        for machines in new_machine_list:
-            # Trimming is done to get just machine name
-            # eg:- machine_name:- prsurve-40-ocs-43-kbrvf-worker-us-east-2b-nlgkr
-            # After trimming:- prsurve-40-ocs-43-kbrvf-worker-us-east-2b
-            if re.match(machines.name[:-6], machine_name):
-                new_machine_name = machines.name
-        machineset_name = machine.get_machineset_from_machine_name(new_machine_name)
-        log.info("Waiting for new worker node to be in ready state")
-        machine.wait_for_new_node_to_be_ready(machineset_name)
-        new_node_name = node.get_node_from_machine_name(new_machine_name)
-        log.info("Adding ocs label to newly created worker node")
-        node_obj = ocp.OCP(kind='node')
-        node_obj.add_label(
-            resource_name=new_node_name,
-            label=constants.OPERATOR_NODE_LABEL
-        )
-        log.info(
-            f"Successfully labeled {new_node_name} with OCS storage label"
-        )
+        if config.ENV_DATA['platform'].lower() == constants.AWS_PLATFORM:
+            if config.ENV_DATA['deployment_type'] == 'ipi':
+                node.delete_and_create_osd_node_aws_ipi(osd_node_name)
+
+            elif config.ENV_DATA['deployment_type'] == 'upi':
+                node.delete_and_create_osd_node_aws_upi(osd_node_name)
+            else:
+                log.warning("'deployment_type' value is not valid")
```

Comment:
In this case, it makes sense to make the test fail because of invalid ocs-ci configuration, eg.:

```
pytest.fail(f"ocs-ci config 'deployment_type' value '{config.ENV_DATA['deployment_type']}' is not valid, results of this test run are all invalid.")
```

We want to make sure this is noticed immediately, and we also want to communicate that test results have no real meaning in such case.